client/my-sites/domains/domain-management/dataviews/use-bulk-action-notice.tsx

```diff
@@ -0,0 +1,114 @@
+import { JobStatus } from '@automattic/data-stores';
+import { StatusPopover } from '@automattic/domains-table/src/status-popover';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import wpcomRequest from 'wpcom-proxy-request';
+import { successNotice, errorNotice } from 'calypso/state/notices/actions';
+import { useDomainsDataViewsContext } from './use-context';
+
+const getSuccessMessage = ( job: JobStatus, translate: ( original: string ) => string ) => {
+	if ( job.action !== 'set_auto_renew' ) {
+		return job.success.length > 1
+			? translate( 'Your domains have been updated.' )
+			: translate( 'Your domain has been updated.' );
+	}
+
+	// If the user tried to enable auto-renew:
+	if ( job.params.auto_renew ) {
+		return job.success.length > 1
+			? translate( 'Automatic renewal has been enabled for your domains.' )
+			: translate( 'Automatic renewal has been enabled for your domain.' );
+	}
+
+	// If the user tired to disable auto-renew:
+	return job.success.length > 1
+		? translate( 'Automatic renewal has been disabled for your domains.' )
+		: translate( 'Automatic renewal has been disabled for your domain.' );
+};
+
+const getFailureMessage = ( job: JobStatus, translate: ( original: string ) => string ) => {
+	if ( job.action !== 'set_auto_renew' ) {
+		return job.failed.length > 1
+			? translate( 'Some domain updates were not successful.' )
+			: translate( 'Your domain update has failed.' );
+	}
+
+	if ( job.params.auto_renew ) {
+		return job.failed.length > 1
+			? translate( 'Enabling automatic renewal has failed for your domains.' )
+			: translate( 'Enabling automatic renewal has failed for your domain.' );
```

Comment:
It's not a blocker for merging this; I just wanted to mention that the `translate` method has plural support.

Example:
```
translate(
    'Enabling automatic renewal has failed for your domain.',
    'Enabling automatic renewal has failed for your domains.',
    { count: job.failed.length }
)
```

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/100576/commits/453c1d245b1f6a3ff3fa67cbcd04a7e91cd6fe02